keras_rs/src/layers/embedding/base_distributed_embedding.py

```diff
@@ -200,6 +270,24 @@ def call(
         weights: Optional[types.Nested[types.Tensor]] = None,
         training: bool = False,
     ) -> types.Nested[types.Tensor]:
+        """Lookup features in embedding tables and apply reduction.
+
+        Args:
+            inputs: A nested structure of 1D tensors to embed, or 2D tensor to
```

Comment:
nit: 2D tensor**s**.

Reply:
Will fix, but what is supported on JAX for dense tensors? On TF, they have to either be 1D or be `(batch_size, 1)`.

Reply:
Meaning that with dense tensors you can't actually do any reduction.

Reply:
`(batch_size, N)`.  It can take a full dense tensor, with reduction.  Same as if it were ragged with each row the same width.

Reply:
That makes sense, I'll try to add some pre/post processing in `call` to support that on TF.